_template/README.md

```diff
@@ -10,7 +10,8 @@ github.com/kubeedge/mappers-go$ make template
 Please input the mapper name (like 'Bluetooth', 'BLE'): test
 ```
 A folder named as you input will be generated in the directory "mappers". The file tree is as below:
-mapper
+```
+mappers/test
```

Comment:
it seems that no need to change the root dirctory from "mapper" to "mappers/test".

Reply:
ok